dotnet/src/SemanticKernel.Abstractions/SemanticKernel.Abstractions.csproj

```diff
@@ -29,6 +29,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Microsoft.SemanticKernel.Core" />
     <InternalsVisibleTo Include="SemanticKernel.UnitTests" />
+    <InternalsVisibleTo Include="Microsoft.SemanticKernel.Experimental.Prompty" />
```

Comment:
This shouldn't be needed, can you explain why it is?

Reply:
This is to access the constructor of `KernelFunction` class, which is marked as internal

Reply:
Reverted